test/e2e/edgedevice_test.go

```diff
@@ -245,6 +246,20 @@ func (e *edgeDeviceDocker) Register(cmds ...string) error {
 		return err
 	}
 
+	if name, exists := os.LookupEnv("TEST_RPM"); name != "" && exists {
+		fp, err := archive.Tar(name, archive.Gzip)
+		if err != nil {
+			return err
+		}
+		err = e.cli.CopyToContainer(ctx, e.name, "/var/tmp", fp, types.CopyToContainerOptions{AllowOverwriteDirWithFile: true})
+		if err != nil {
+			return err
+		}
+		if _, err = e.Exec(fmt.Sprintf("dnf remove -y flotta-agent-race && dnf install -y /var/tmp/%s", filepath.Base(name))); err != nil {
```

Comment:
shouldn't `TEST_RPM` include multiple rpms?
in which cases would we like to use `TEST_RPM`?

Reply:
Actually it can. I know it doesn't specifically says, what RPM it is, so it can be any, but I remove the flotta-agent, so I assume it's for flotta-agent. Do you suggest do it more generic?
TEST_RPM is used to test your local changes as:
```FOCUS="XXX" SKIP_TEST_IMAGE_PULL=true TEST_IMAGE=quay.io/project-flotta/edgedevice:my TEST_RPM="/home/omachace/rpmbuild/RPMS/x86_64/flotta-agent-race-0.1.0-1.fc34.x86_64.rpm" make integration-test
```
Or you can run manual workflow job, if you specify link to your RPM build by workflow job: like this one: https://github.com/project-flotta/flotta-device-worker/suites/6671226059/artifacts/252640845